client/my-sites/checklist/wpcom-checklist/checklist-prompt/index.jsx

```diff
@@ -31,31 +32,31 @@ export class ChecklistPrompt extends Component {
 	};
 
 	render() {
-		const { translate, taskList, children } = this.props;
+		const { translate, taskList, children, promptTaskId } = this.props;
 
 		const childrenArray = Children.toArray( children );
 
 		const firstIncomplete = taskList.getFirstIncompleteTask();
 		const isFinished = ! firstIncomplete;
+		const promptTask = promptTaskId && taskList.get( promptTaskId );
+		const theTaskId = promptTask ? promptTask.id : firstIncomplete.id;
 
 		return (
 			<div className="checklist-prompt">
 				{ isFinished ? (
-					<>
-						<ChecklistPromptTask
-							description={ translate(
-								'We did it! You have completed {{a}}all the tasks{{/a}} on our checklist.',
-								{
-									components: {
-										a: <a href={ `/checklist/${ this.props.siteSlug }` } />,
-									},
-								}
-							) }
-							title={ translate( 'Your site is ready to share' ) }
-						/>
-					</>
+					<ChecklistPromptTask
+						description={ translate(
+							'We did it! You have completed {{a}}all the tasks{{/a}} on our checklist.',
```

Comment:
This is not a big deal, but what do folks think about changing the possessive article to `'your'`?

_Our checklist_ insinuates a checklist we've imposed upon or given to the user, but I feel the checklist belongs to the user or at least the user's site.

Reply:
@danhauk ^^

Reply:
Updated!